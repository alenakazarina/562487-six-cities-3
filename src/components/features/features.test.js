import React from 'react';
import renderer from 'react-test-renderer';
import Features from './features';

const TYPE = `hotel`;
const BEDROOMS = [2, 1];
const MAX_ADULTS = [3, 1];

describe(`Features`, () => {
  it(`should render Features with correct plural nouns`, () => {
    const tree = renderer.create(
        <Features
          type={TYPE}
          bedrooms={BEDROOMS[0]}
          maxAdults={MAX_ADULTS[0]}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it(`should render Features with correct singular nouns`, () => {
    const tree = renderer.create(
        <Features
          type={TYPE}
          bedrooms={BEDROOMS[1]}
          maxAdults={MAX_ADULTS[1]}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
