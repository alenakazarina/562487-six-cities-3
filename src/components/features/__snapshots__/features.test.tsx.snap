// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features should render Features with correct plural nouns 1`] = `
<ul
  className="property__features"
>
  <li
    className="property__feature property__feature--entire"
  >
    hotel
  </li>
  <li
    className="property__feature property__feature--bedrooms"
  >
    2 Bedrooms
  </li>
  <li
    className="property__feature property__feature--adults"
  >
    Max 
    3 adults
  </li>
</ul>
`;

exports[`Features should render Features with correct singular nouns 1`] = `
<ul
  className="property__features"
>
  <li
    className="property__feature property__feature--entire"
  >
    hotel
  </li>
  <li
    className="property__feature property__feature--bedrooms"
  >
    1 Bedroom
  </li>
  <li
    className="property__feature property__feature--adults"
  >
    Max 
    1 adult
  </li>
</ul>
`;
